otter/test/test_controller.py

```diff
@@ -1079,3 +1080,26 @@ def test_audit_log_scale_down(self):
             policy_id=None, webhook_id=None,
             active_capacity=3, desired_capacity=5, pending_capacity=2,
             audit_log=True)
+
+    def test_real_convergence(self):
+        """
+        When a tenant is configured to for convergence, the Convergence
+        service's ``converge`` method is invoked, and None is returned.
+        """
+        log = mock_log()
+        converger_mock = mock.Mock()
+        state = GroupState('tenant', 'group-id', "test", [], [], None, {},
+                           False)
+        group_config = {'maxEntities': 100, 'minEntities': 0}
+        policy = {'change': 5}
+        config_data = {'convergence-tenants': ['tenant']}
+
+        self.addCleanup(set_converger, get_converger())
+        set_converger(converger_mock)
+
+        result = controller.converge(log, 'txn-id', group_config, self.group,
+                                     state, 'launch', policy,
+                                     config_value=config_data.get)
+        self.assertIs(result, None)
+        converger_mock.start_convergence.assert_called_once_with(
+            log, 'tenant', 'group-id', 5, 'launch')
```

Comment:
Could you please also check if `execute_launch_config` is not called in this case? Also, will be nice to do vice versa in other tests :) i.e. `converger` not called on regular tenants.